third-party/chromium/data/verify_signed_data/annotate_test_data.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
```

Comment:
I don't think that the stuff generated by `gen-test-signatures.sh` is really suitable to take over the job of the Chromium test files. It was the most expedient way I could think of to provide a cross-product of signature algorithm, key size, and digest algorithm for a fixed set of inputs. The Chromium test files, on the other hand, are much more flexible, and they allow for comments easily, they are clumsy to assemble en masse (as would be needed for the above cross-product). The `tests/test-data` scheme doesn't have a good way to express things like this:

> This uses the same test data as ecdsa-prime256v1-sha512.pem, HOWEVER the
algorithm OID for the public key has been changed from id-ecPublicKey
(1.2.840.10045.2.1) to id-ecMQV (1.3.132.1.13). This test should fail because the public key's algorithm does not match that of
the signature algorithm.

but `ecdsa-prime256v1-sha512-using-ecmqv-key.pem` does that pretty well.

I'm not vigorously in favor of the Chromium format per se, but it also seemed like overkill for what I wanted to do with SPKI. I don't really know what best practices are for maintaining test files of delicately set up binary inputs and outputs. If the Chromium team has already been around this particular block and decided that their PEM format is The Right Choice, then I can re-work the spki tests to use that format. Or, we can come up with yet another format that supersedes both.

(Also, could just reimplement the Python in Rust and have it live in `examples` as a dev tool.)